src/test/java/io/jenkins/plugins/forensics/git/blame/GitBlamerTest.java

```diff
@@ -295,13 +319,14 @@ private RawText createResultSize(final int size) {
     private void stubResultForIndex(final BlameResult result, final int index) {
         int line = index + 1;
         Mockito.when(result.getSourceAuthor(index)).thenReturn(new PersonIdent(NAME + line, EMAIL + line));
-        RevCommit commit = Mockito.mock(RevCommit.class);
+        RevCommit commit = createCommit();
         Mockito.when(result.getSourceCommit(index)).thenReturn(commit);
     }
 
     private void verifyResult(final FileBlame request, final int line) {
         assertThat(request.getEmail(line)).isEqualTo(EMAIL + line);
         assertThat(request.getName(line)).isEqualTo(NAME + line);
         assertThat(request.getCommit(line)).isNotBlank().isNotEqualTo(EMPTY); // final getter
+        assertThat(request.getTime(line)).isNotEqualTo(EMPTY_TIME);
```

Comment:
Would it be possible to check for the exact value since you found a way to create a stub with concrete values? (line 329 and 330)